examples/dash/dash-saml/readme.md

```diff
@@ -0,0 +1,57 @@
+# Dash SAML authentication
+
```

Comment:
Please add a subtitle for the app, e.g.

```
# Dash SAML authentication

Demonstration app for SAML integration with Dash

> [!CAUTION]
...
```

Reply:
https://github.com/ploomber/doc/pull/292/commits/79a934ead9216902bad322f9710bcef8725f94d4